modules/statics/src/coins.ts

```diff
@@ -9275,6 +9275,424 @@ export const coins = CoinMap.fromCoins([
     '0x152b9d0fdc40c096757f570a51e494bd4b943e50',
     UnderlyingAsset['avaxc:btc']
   ),
+  // Begin FTX missing AVAXC tokens
+  avaxErc20(
+    'c98de371-69a7-4f06-9a6e-810f0fc2b0bc',
+    'avaxc:yeti',
+    'Yeti Finance',
+    18,
+    '0x77777777777d4554c39223c354a05825b2e8faa3',
+    UnderlyingAsset['avaxc:yeti']
+  ),
+  avaxErc20(
+    '0e477e9c-cb81-4988-9f15-a8b5f186932a',
+    'avaxc:spell',
+    'Spell Token',
+    18,
+    '0xce1bffbd5374dac86a2893119683f4911a2f7814',
+    UnderlyingAsset['avaxc:spell']
+  ),
+  avaxErc20(
+    '35a0282a-a6f2-4df5-95ad-13e1316441ae',
+    'avaxc:yusd',
+    'YUSD Stablecoin',
+    18,
+    '0x111111111111ed1d73f860f57b2798b683f2d325',
+    UnderlyingAsset['avaxc:yusd']
+  ),
+  avaxErc20(
+    '005cae58-7917-4288-bf24-60752d193dbc',
+    'avaxc:yusdcrv-f',
+    'Curve.fi Factory Plain Pool: YUSD Pool',
+    18,
+    '0x1da20ac34187b2d9c74f729b85acb225d3341b25',
+    UnderlyingAsset['avaxc:yusdcrv-f']
+  ),
+  avaxErc20(
+    'a16665e2-cb8a-4dcf-876a-a82e8d0a187c',
+    'avaxc:ecd',
+    'Echidna Token',
```

Comment:
```suggestion
    'EDA Token',
```
The other ones above follow the pattern of using the words after the `:`. Any reason for it being different here?